HealthcareSystem/PatientWebApp/Controllers/Adapter/RequestAdapter.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using RestSharp;
+
+namespace PatientWebApp.Controllers.Adapter
+{
+    public class RequestAdapter
+    {
+        public static ContentResult SendRequestWithoutBody(string baseUrl, string resource, Method method)
+        {
+            var client = new RestClient(baseUrl);
+            var request = new RestRequest(resource, method);
+            var response = client.Execute(request);
+
+            var contentResult = new ContentResult();
+            contentResult.Content = response.Content;
+            contentResult.ContentType = "application/json";
```

Comment:
Why not take the ContentType from the response as well?

Reply:
I didn't know that response has a field ContentType. I'll fix that for the next sprint.